trax/data/bert_inputs_test.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+# Copyright 2020 The Trax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Tests for trax.data.bert_inputs."""
+
+import os
+
+from functools import partial
+
+from absl.testing import absltest
+
+import numpy as np
+
+from trax.data import bert_inputs
+from trax.models.research.bert import download_model_if_model_name
+
+
+class BertInputsTest(absltest.TestCase):
```

Comment:
These tests can hopefully be moves as-is to trax/data/tf_inputs_test

Reply:
because pipeline has changed too much i created new tests in tf_inputs_test and deleted this file